plugins/machina-tracker.js

```diff
@@ -30,13 +30,17 @@ machinaTracker.events = [];
 machinaTracker.setup = () => {
   $('<style>').prop('type', 'text/css').html('@include_string:machina-tracker.css@').appendTo('head');
 
-  var iconImage = '@include_img:images/marker-machina.png@';
-
-  machinaTracker.icon = L.icon({
-    iconUrl: iconImage,
-    iconSize: [26, 32],
-    iconAnchor: [12, 32],
-  });
+  var iconImage = '@include_img:images/marker-machina.svg@';
+
+  machinaTracker.icon = new (L.Icon.Default.extend({
+    options: {
+      iconUrl: iconImage,
+      iconRetinaUrl: iconImage,
+      iconSize: [26, 32],
+      iconAnchor: [12, 32],
```

Comment:
Any comment on why you want to specify different sizes than defaults here?

It makes the icon smaller than the Enl/Res tracker.
And the values are now same as default, then remove them and keep the default in use - there's no need to specify it again and will result in more work later when we'll want to change the size of all these tracker markers.